### ISSUE TITLE
Checklist For Return

### ISSUE THREAD

**Issue by magico13** [magico13/KCT#126]
Here's a list of smaller things that I should do when I come back to working on this:
- Option to prevent rollout when reconditioning is active [possibly relevant link](http://forum.kerbalspaceprogram.com/index.php?/topic/83342-105-kerbal-construction-time-132-mar-7-2016-unrapid-planned-assembly/&page=84#comment-2470580)
- KAC alarm upgrades
- part/module specific modifiers
- editing process overhaul (RF tanks, flipping in SPH, saving edits)

**Comment by Baleine82** [comment]
About the rollout, you can indeed rollout when the pad is reconditionning, but you can't launch until reconditioning is finished

**Comment by magico13** [comment]
This is just a checklist reiterating other things that already exist, so it's redundant. The first and third things are either completed or not really an issue. The alarm upgrades is a separate issue #104 and the editing process overhaul is a few other separate things.
